### ISSUE TITLE
Check crashes when a data-file is missing

### ISSUE THREAD

**Issue by sigio** [restic/restic#275]
Steps to reproduce:

```
restic init -t /tmp/backup
restic backup /some/path
rm /tmp/backup/data/f5/f5ae1784d87f0235b96eaa9ef0334e9b6e3faff577f6832c1961b930db74cd2b
restic check

enter password for repository: 
Create exclusive lock for repository
Load indexes
Check all packs
pack f5ae1784d87f0235b96eaa9ef0334e9b6e3faff577f6832c1961b930db74cd2b: does not exist
Check snapshots, trees and blobs
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4def54]

goroutine 88 [running]:
runtime.panic(0x852320, 0xb1bb13)
	/usr/lib/go/src/pkg/runtime/panic.c:279 +0xf5
```

**Comment by fd0** [comment]
Thanks for the bug report, I'll take a look.

**Comment by fd0** [comment]
Btw: The project is called "restic", not "rustic" ;)

**Comment by fd0** [comment]
This issue is fixed since 5c39abfe53c61ce3594224f6cd2d4a0a0486fc75, thanks for your report.
